vprof/code_heatmap.py

```diff
@@ -91,7 +91,8 @@ def _calc_skips(self, heatmap, num_lines):
             skips.append((prev_line, num_lines - prev_line))
         return skips
 
-    def _skip_lines(self, src_code, skip_map):
+    @staticmethod
```

Comment:
Technically `_skip_lines` is static method, but I find writing `ClassName.static_method()` more verbose than `self.static_method()`.